source/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,156 @@
+// Variables
+
+$light:  #febbbb;
+$medium: #fe9090;
+$dark:   #ff5c5c;
+$dark-v1: #c84848;
+$text-color: #999;
+$cta: #00c5a7;
+$card-font-small: 15px;
+
+
+// Global Styles
+/*#press {
+  background: $light;
+  background: linear-gradient(135deg, $light 0%,$medium 45%,$dark 100%);
+  font-size: 10px;
+  padding: 2em;
+}
+.full-screen {
+  height: 100vw;
+  position: relative;
+}*/
+
+
+// General content
+.title {
+  text-align: center;
+  font-weight: 100;
+}
+.divider {
+  border-bottom: 1px solid $light;
```

Comment:
wanted to give the `hr` another color, but you're right it makes it more complicated than what it should be.